.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.0","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/harrybaxter/Library/Application Support/IdeaIC2019.3/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/harrybaxter/Library/Application%20Support/IdeaIC2019.3/Scala/launcher/sbt-launch.jar"]}
```

Comment:
what is this? it seems like it just gets auto added in for some reason, it did when I opened the repo on my machine.

Reply:
I never even noticed that. Not sure what it is